usaspending_api/search/v2/views/spending_by_category_views/spending_by_category.py

```diff
@@ -150,7 +150,7 @@ def build_elasticsearch_search_with_aggregations(self, filter_query: ES_Q) -> Op
             group_by_agg_key_values = {"order": {"sum_field": "desc"}}
         else:
             # Get count of unique buckets; terminate early if there are no buckets matching criteria
-            bucket_count = get_number_of_unique_terms(filter_query, f"{self.category.agg_key}.hash")
+            bucket_count = get_number_of_unique_terms(filter_query, f"{self.category.agg_key}")
```

Comment:
This needs to have the `.hash` on the end since the method `get_number_of_unique_terms` simply runs cardinality aggregation on the field you pass.